src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios.md

```diff
@@ -4,14 +4,16 @@ excerpt: ''
 hideFromSidebar: false
 ---
 
-k6 v0.27.0 introduced the concept of scenarios: configurable modes of scheduling VUs
-and iterations, which can be used to model diverse traffic patterns in load tests.
+> ### 🎉 New in v0.27.0
+>
+> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,
+> existing scripts and configurations will continue to work as before. For a list of breaking changes,
+> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
 
-This is an optional feature and existing scripts and options will continue to work as before
-(with a few minor breaking changes, as mentioned
-in the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
+## Benefits
```

Comment:
I don't think this should be a header, since we haven't explained what scenarios are at this point. The paragraph below is both an introduction to the feature and a list of its benefits, so I think we're fine with just the preamble of "There are several benefits...".